packages/teams-js/src/private/teams.ts

```diff
@@ -82,4 +85,77 @@ export namespace teams {
   export function isSupported(): boolean {
     return runtime.supports.teams ? true : false;
   }
+
+  export namespace fullTrust {
+    export namespace joinedTeams {
+      /**
+       * @hidden
+       * Hide from docs
+       * ------
+       * Allows an app to retrieve information of all user joined teams
+       *
+       * @param teamInstanceParameters - OPTIONAL Flags that specify whether to scope call to favorite teams
+       * @returns Promise resolved containing information about the user joined teams or rejected with error
+       */
+      export function getUserJoinedTeams(
+        teamInstanceParameters?: TeamInstanceParameters,
+      ): Promise<UserJoinedTeamsInformation> {
+        return new Promise<UserJoinedTeamsInformation>(resolve => {
+          ensureInitialized();
+          if (!isSupported()) {
+            throw errorNotSupportedOnPlatform;
+          }
+
+          if (
+            (GlobalVars.hostClientType === HostClientType.android ||
+              GlobalVars.hostClientType === HostClientType.teamsRoomsAndroid ||
+              GlobalVars.hostClientType === HostClientType.teamsPhones ||
+              GlobalVars.hostClientType === HostClientType.teamsDisplays) &&
+            !isCurrentSDKVersionAtLeast(getUserJoinedTeamsSupportedAndroidClientVersion)
+          ) {
+            const oldPlatformError: SdkError = { errorCode: ErrorCode.OLD_PLATFORM };
+            throw new Error(JSON.stringify(oldPlatformError));
+          }
+
+          resolve(sendAndUnwrap('getUserJoinedTeams', teamInstanceParameters));
+        });
+      }
+
+      export function isSupported(): boolean {
+        return runtime.supports.teams
+          ? runtime.supports.teams.fullTrust
+            ? runtime.supports.teams.fullTrust.joinedTeams
+              ? true
+              : false
+            : false
+          : false;
+      }
+    }
+
+    /**
+     * @hidden
+     * Hide from docs
+     * ------
+     * Allows an app to get the configuration setting value
+     *
+     * @param key - The key for the config setting
+     * @returns Promise resolved containing the value for the provided config setting or rejected with error
+     */
+    export function getConfigSetting(key: string): Promise<string> {
+      return new Promise<string>(resolve => {
+        ensureInitialized();
+        if (!isSupported()) {
+          throw errorNotSupportedOnPlatform;
+        }
+        resolve(sendAndUnwrap('getConfigSetting', key));
+      });
+    }
+
+    /**
+     * Checks if teams.fullTrust capability is supported currently
+     */
```

Comment:
Can you please change this to:

    /**
     * @hidden
     * Hide from docs
     * ------
     * Checks if teams.fullTrust capability is supported by the host
     * @returns true if the teams.fullTrust capability is enabled in runtime.supports.teams.fullTrust and
     * false if it is disabled
     */